tests/sentry/utils/performance_issues/test_n_plus_one_api_calls_detector.py

```diff
@@ -107,6 +137,57 @@ def test_respects_project_option(self):
 
         assert not detector.is_creation_allowed_for_project(project)
 
+    def test_fingerprints_events(self):
+        event = self.create_event(lambda i: "GET /clients/info")
+        [problem] = self.find_problems(event)
+
+        assert problem.fingerprint == "1-1010-3378cb14bc594ab8c1c32beca224f9c4d0b830aa"
+
+    def test_fingerprints_identical_relative_urls_together(self):
+        event1 = self.create_event(lambda i: "GET /clients/info")
+        [problem1] = self.find_problems(event1)
+
+        event2 = self.create_event(lambda i: "GET /clients/info")
```

Comment:
Not sure if this test is needed if event 1 and event 2 have the same parameter? 
Is this supposed to test if the event fingerprints are consistent? But if fingerprints weren't consistent, then all the tests expecting a fingerprint would be flakey